config/initializers/rateit_app.rb

```diff
@@ -1,2 +1,3 @@
 # frozen_string_literal: true
 # Rateitapp.plugin_manager.add('name_of_your_plugin')
+Rateitapp.oauth_domain = 'accounts.publicradio.org'
```

Comment:
Looks to me like we don't need this file (config/initializers/rateit_app.rb) in the engine at all anymore.